src/Simplex/Chat/Types.hs

```diff
@@ -214,7 +214,8 @@ groupName' GroupInfo {localDisplayName = g} = g
 
 -- TODO when more settings are added we should create another type to allow partial setting updates (with all Maybe properties)
 data ChatSettings = ChatSettings
-  { enableNtfs :: Bool
+  { enableNtfs :: Bool,
+    chatUnread:: Bool
```

Comment:
this is not the right place, it's not a setting. Better to put `unreadChat` into ChatStats (that has unreadCount now)